curriculum/challenges/english/25-front-end-development/quiz-css-animations/66ed8fc9f45ce3ece4053eae.md

```diff
@@ -486,3 +486,464 @@ Which CSS property allows you to set a time before the animation begins?
+
+#### --answer--
+
+That any looping animations only play once
+
+### --question--
+
+#### --text--
+
+What type of animation content can trigger seizures in people with photosensitive epilepsy?
+
+#### --distractors--
+
+Slow fading transitions
+
+---
+
+Background-color changes
+
+---
+
+Single-direction transforms
+
+#### --answer--
+
+Rapid flashing or strobing effects that flash more than three times per second
+
+### --question--
+
+#### --text--
+
+Which property is NOT part of the `animation` shorthand?
+
+#### --distractors--
+
+animation-delay
+
+---
+
+animation-timing-function
+
+---
+
+animation-direction
+
+#### --answer--
+
+animation-transform
+
+### --question--
+
+#### --text--
+
+What does the `@keyframes` rule define?
+
+#### --distractors--
+
+The timing function of an animation
+
+---
+
+The default state of an element
+
+---
+
+The media queries for animations
+
+#### --answer--
+
+The sequence of styles at different points in an animation
+
+### --question--
+
+#### --text--
+
+What is the goal of using the `prefers-reduced-motion` media query in animations?
+
+#### --distractors--
+
+To completely remove all motion from the website
+
+---
+
+To force animations to run at double speed
+
+---
+
+To disable all CSS styles globally
+
+#### --answer--
+
+Use a low-motion approach that respects user settings for comfortable accessibility.
+
+### --question--
+
+#### --text--
+
+In a keyframes rule, what does `100%` represent?
+
+#### --distractors--
+
+The start of the animation
+
+---
+
+The halfway point
+
+---
+
+The easing function
+
+#### --answer--
+
+The end of the animation
+
+### --question--
+
+#### --text--
+
+Which property controls the pace of an `animation` over its duration?
+
+#### --distractors--
+
+`animation-duration`
+
+---
+
+`animation-delay`
+
+---
+
+`animation-iteration-count`
+
+#### --answer--
+
+`animation-timing-function`
+
+### --question--
+
+#### --text--
+
+What should developers consider when implementing animations to maintain accessibility?
+
+#### --distractors--
+
+Using only JavaScript for all animations
+
+---
+
+Adding as many animations as possible for engagement
+
+---
+
+Using only large, unexpected movements
+
+#### --answer--
+
+Use purposeful animation, avoid sudden large motions, provide controls, and implement `prefers-reduced-motion`
+
+### --question--
+
+#### --text--
+
+Which of the following is the correct syntax to slide an element in from the left?
+
+#### --distractors--
+
+```js
+@keyframes slide-in {
+  0    { transform: translate(-100%); }
+  100  { transform: translate(0); }
+}
+```
+
+---
+
+```js
+@keyframes slide-in {
+  from { translateX(-100%); }
+  to   { translateX(0); }
+}
+```
+
+---
+
+```js
+@keyframes slide-in {
+  start { transform: moveX(-100%); }
+  end   { transform: moveX(0); }
+}
+```
+
+#### --answer--
+
+```js
+@keyframes slide-in {
+  0%   { transform: translateX(-100%); }
+  100% { transform: translateX(0); }
+}
+```
```

Comment:
the wrong language is specified here. it should be css not js.
Also, css examples should use formatting consistent with review pages and lectures like this

```css
@keyframes slide-in {
  0% {
    transform: translateX(-100%);
  }
  100% {
    transform: translateX(0);
  }
}

```

please fix all options